src/services/currencyAdapter.service.ts

```diff
@@ -0,0 +1,68 @@
+import { BadRequestError } from '../errors/Errors';
+
+export enum AllowedCryptoCurrencies {
+  Bitcoin = 'BTC',
+  Ethereum = 'ETH',
+  Litecoin = 'LTC',
+  BitcoinCash = 'BCH',
+  Ripple = 'XRP',
+  Tether = 'USDT',
+  USDC = 'USDC',
+  Tron = 'TRX',
+  Cardano = 'ADA',
+  BinanceCoin = 'BNB',
+}
+
+export enum AllowedFiatCurrencies {
+  Euro = 'EUR',
+  USDollar = 'USD',
+}
+
+export class CurrencyAdapter {
```

Comment:
If everything is static, why don't just use functions?

Reply:
Ah, understood. It's better to add this outside of services` 👍🏽